### DISCUSSION TITLE
Help needed with Deepgram Self Hosted setup -  'transaction_key': 'deprecated' error

### DISCUSSION THREAD

**Discussion by devalbham** [deepgram/community#1021] [General help]
Hello, 

I have been using Deepgram Self-hosted setup for live streaming transcription successfully, however, very recently, it has stopped transcribing. I am getting below response/error:

Full response from Deepgram: {'type': 'Metadata', 'transaction_key': 'deprecated', 'request_id': 'e0204251-338f-406a-98f5-0568b6743269', 'sha256': '72cc007a5448ec0f727c2dda1c58e4be2a0a32c69a41ac71af725d0c0253cfff', 'created': '2024-12-09T15:30:36.051Z', 'duration': 0.0, 'channels': 0}
Unexpected or empty response from Deepgram: {'type': 'Metadata', 'transaction_key': 'deprecated', 'request_id': 'e0204251-338f-406a-98f5-0568b6743269', 'sha256': '72cc007a5448ec0f727c2dda1c58e4be2a0a32c69a41ac71af725d0c0253cfff', 'created': '2024-12-09T15:30:36.051Z', 'duration': 0.0, 'channels': 0}
Connection with Deepgram closed unexpectedly

After this websocket connection is closed and I do not receive any transcription. My backend is written in FastAPI receiving requests over websockets from clients and then forwarding them to Deepgram API and to Deepgram self-hosted Engine for transcription. 

For reference, request ID is: e0204251-338f-406a-98f5-0568b6743269

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
